mart/configs/experiment/CIFAR10_CNN_Adv.yaml

```diff
@@ -1,7 +1,7 @@
 # @package _global_
 
 defaults:
-  - /attack@callbacks.adversarial_training.adversary: classification_eps1.75_fgsm
+  - /attack@callbacks.adversarial_training.train_adversary: classification_eps1.75_fgsm
```

Comment:
Why the change?

Reply:
We don't run adversary in validation in this experiment.